src/content/docs/journey-performance/slo-tutorial.mdx

```diff
@@ -0,0 +1,296 @@
+* Standardize reliability: Cross-organizational teams have a unified, transparent view of service reliability, and can better comply with customer-facing SLAs. SLO compliance metrics and error budgets give organizations a way to report on reliability and implement changes across applications, infrastructure, and teams in a cohesive fashion.
+
+Let's get started:
+
+<Steps>
+    <Step>
+
+    ## Select service level indicators:
+
+    While there are a large amount of SLIs you could use to define your front end experience benchmarks, the following are some we specifically reccomend. Each collapser has an explanation of when you should choose that SLI and a corresponding NRQL query (which you will use in step 2).
+
+    For now, just select one of the following:
+
+    **SLIs for APM services instrumented with the New Relic agent**:
+    Based on `Transaction` events, these SLIs are the most common for request-driven services:
+
+    <CollapserGroup>
+    <Collapser
+        className="freq-link"
+        id="service-availability"
+        title="Service success"
+    >
+        Service success is the ratio of the number of successful responses to the number of all requests. This effectively is an error rate, but you can filter it down, for example removing expected errors.
+
+        **Valid events fields**
+
+        ```sql
+        FROM: Transaction
+        WHERE: entityGuid = '{entityGuid}'
+        ```
+
+        Where `{entityGuid}` is the service's GUID.
+
+        **Bad events fields**
+
+        ```sql
+        FROM: TransactionError
+        WHERE: entityGuid = '{entityGuid}' AND error.expected IS FALSE
+        ```
+
+        Where `{entityGuid}` is the service's GUID.
+    </Collapser>
+
+    <Collapser
+        className="freq-link"
+        id="service-latency"
+        title="Service latency"
+    >
+        A latency SLI measures the proportion of valid requests that were served faster than the threshold established as a good experience.
+
+        In order to determine that duration threshold, check how the service has been performing in the past weeks, and use that result as a realistic and achievable baseline. Afterwards, you can iterate on the SLI threshold, and align it with a more ambitious performance.
+
+        To select an appropriate value for the duration condition, one typical practice is to select the 95 percentile duration of the responses for the last 7 or 15 days. Find this duration threshold using the [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/), and use it to determine what you consider to be good events for your SLI:
```

Comment:
Statements like "one typical process" always feel a little dry to me personally. I always try to replace them with more action oriented or relatable statements, like "you can try selecting" instead of "one typical practice is to select".